pkg/api/bzzchunk.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package api
+
+import (
+	"bytes"
+	"errors"
+	"io"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/ethersphere/bee/pkg/jsonhttp"
+	"github.com/ethersphere/bee/pkg/storage"
+	"github.com/ethersphere/bee/pkg/swarm"
+	"github.com/gorilla/mux"
+)
+
+func (s *server) chunkUploadHandler(w http.ResponseWriter, r *http.Request) {
+	addr := mux.Vars(r)["addr"]
+	ctx := r.Context()
+
+	address, err := swarm.ParseHexAddress(addr)
+	if err != nil {
+		s.Logger.Debugf("bzz-chunk: parse chunk address %s: %v", addr, err)
```

Comment:
Alongside these Debug log lines, I would add Error log lines that are for users who enable only Error log level, with messages that do not contain any internal implementation details. Debug log levels are for developers debugging internals.

Reply:
Thanks for adding the Error log message. Could you add similar to the other places in these two new handlers? I am not sure if it is needed for io.Copy error, as that error is usually client or network related and without much significance to the server.